src/relay/backend/interpreter.cc

```diff
@@ -124,6 +125,38 @@ struct Stack {
   };
 };
 
+/*! \brief A representation of the interpreter state which can be passed back to Python. */
+class InterpreterState;
+
+/*! \brief Interpreter state container. */
+class InterpreterStateNode : public Node {
+ public:
+  using Frame = tvm::Map<Var, Value>;
```

Comment:
Should we use different names for `Frame` and `Stack` since they are declared as structs above?

Reply:
They are suppose to be the same, just converted to TVM layout so that they can be read by Python.